test/lib/server/services/log/LogService.test.js

```diff
@@ -20,13 +20,16 @@ const { expect } = chai;
 
 module.exports = () => {
     let logMock;
+    let rootLogs;
 
     beforeEach(async () => {
         logMock = {
             title: 'Yet another log',
             text: 'This is the text field of yet another log.',
             tags: [],
         };
+
+        rootLogs = await logService.getAllRootLogs(10, 0);
```

Comment:
in before each, you set up things that should be done for every tests, for example setting mock data.
Getting rootLogs is actually part of your tests, so you should write it in your tests when it applies.

Reply:
done